DoSomethingArchive/legacy-website
# Gets rid of all Rogue conditional flags since Rogue is always on

#### What's this PR do?
- Gets rid of Rogue conditional logic because Rogue will never be turned off!

#### How should this be reviewed?
- Make sure all the logic is still working that you can: 
  - Signup
  - Reportback
  - Update quantity/why participated
  - Gallery is working
  - Gallery pagination is working
  - Kudos / reactions are working (and are persisted when you refresh the page) 
  - User profile page and is working and all data shows up as expected 
  - @sbsmith86 does anything with competitions need to be tested for this?
  - Did I miss anything? 

#### Relevant tickets
Fixes the issue in this [Pivotal Card](https://www.pivotaltracker.com/n/projects/2019429/stories/151415698).

#### Checklist
